ci-operator/step-registry/baremetal/lab/agent/gather/baremetal-lab-agent-gather-commands.sh

```diff
@@ -28,10 +28,6 @@ SSHOPTS=(-o 'ConnectTimeout=5'
 
 echo "Trying to gather agent logs on the host ${HOST_ID}"
 
-if [ "$CLUSTER_WIDE_PROXY" == "true" ] || [ "$DISCONNECTED" == "true" ]; then
- source "${SHARED_DIR}/proxy-conf.sh"
-fi
-
 if ssh "${SSHOPTS[@]}" core@api."${NODE_ZERO}" agent-gather -O >"${ARTIFACT_DIR}"/agent-gather.tar.xz; then
```

Comment:
does this work on disconnected scenarios?

Reply:
Yes, it's working in disconnected scenarios. I have also updated the name to API_NODE_ZERO.

Reply:
@mhanss per my understanding you reach the API endpoint, i.e., api.ci-op-XXXXX.$BASE_DOMAIN.

The API endpoint has the IP of the load balancer. In disconnected environments, this IP should be a private IP like 192.168.x.y (the IP of the interface connected to the machines network via br-int). 
Is this correct?